src/Kunstmaan/AdminBundle/Resources/views/MultiDomainAdminTwigExtension/widget.html.twig

```diff
@@ -0,0 +1,17 @@
+{% set switchedHost = get_switched_host() %}
```

Comment:
We can not put this in the MultiDomainBundle because the twig function needs to be available also when the MultiDomainBundle is not enabled.